src/Components/Http/Installer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Laravel Zero.
+ *
+ * (c) Nuno Maduro <enunomaduro@gmail.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace LaravelZero\Framework\Components\Http;
+
+use LaravelZero\Framework\Components\AbstractInstaller;
+
+/**
+ * @internal
+ */
+final class Installer extends AbstractInstaller
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected $name = 'install:http';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $description = 'Http: Manage web requests using a fluent HTTP client';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function install(): void
+    {
+        $this->require('guzzlehttp/guzzle "^6.3.1"');
```

Comment:
I think you can install both at the same time.

Reply:
This does seem to work if they are space separated, but the other installers seem to do them separately.

I'm happy to change this if you think it would be better (I know using combining them does make it a bit faster).